vipps-partner-api.md

```diff
@@ -260,23 +263,87 @@ We are working on simplifying this in the future.
+In order to get approved to have a product ("Vipps på nett", "Vipps Login", etc)
+with Vipps the merchant must first have a formal agreement with Vipps, called
+the Merchant Agreement (MA).
 
-The state flow can be described as follows.
+Merchants need both a Merchant Agreement (MA) and a Product Order (PO) to
+be able to accept Vipps payments:
+* MA: An agreement between the merchant and Vipps, signed with BankID.
+  The MA contains information about all direct and indirect owners, any
+  politically exposed persons, etc. The MA is signed with BankID.
+* PO: This is an order for "Vipps på nett", "Vipps Login", etc. The merchant
+  must provide some information about thew use, whether the3 cardholder is
+  present, etc. The PO is not signed with BankID.
 
-![Merchant agreement flow](images/merchant-agreement-flow.png)
+A merchant may order a Vipps product (submit a PO) with or without a Merchant Agreement (MA).
 
-The merchant will now see the prefilled product order waiting for them in the portal under "Produktbestillinger klare til innsendelse" in their home screen. (see highlighted area in image)
-
-![example-portal-with-preorder](images/example-portal-with-preorder-waiting.png)
-
-Alternatively the URL created in the initial prefill can be reused for up to 15 days.
+**Please note:** The only method Vipps has to verify that a user has the right
+to sign a MA for a merchant is by using data from
+[Brønnøysundregistrene](https://brreg.no).
+It is therefore a requirement that the user logging in on
+[portal.vipps.no](https://portal.vipps.no)
+is registered as chairman of the board ("styreleder") or CEO ("daglig leder").
+
+#### Scenario 1: The merchant does not have a Merchant Agreement
+
+1. The partner prefills the PO using
+   [`POST:/products/orders`](https://vippsas.github.io/vipps-developer-docs/api/partner#tag/Vipps-Product-Orders/operation/orderProduct)
+   and gets a link to the prefilled PO on
+   [portal.vipps.no](https://portal.vipps.no).
+2. The merchant user uses the link and logs in with BankID on
+   [portal.vipps.no](https://portal.vipps.no).
+3. Based on the prefilled data and the BankID login,
+   the merchant is presented with the form(s) for the MA.
+4. The merchant completes and submits the MA for processing.   
+5. The merchant is presented with the prefilled PO,
+   checks the details in the PO and submits it.
+6. Vipps processes the PO and sends both the merchant and partner an
+   email when done. The partner can also check with the API:
+   [`GET:/merchants/{orgno}`](https://vippsas.github.io/vipps-developer-docs/api/partner#tag/Merchants/operation/getMerchant).
```

Comment:
Have you tested this. Wouldn't this just continue with scenario 2, since it would be in a state where the MA is being processed.

Reply:
Nope, I have not tested. That's what I need help with.